src/direct_commands/claim.ts

```diff
@@ -0,0 +1,34 @@
+import { ChannelType, Client, Message } from "discord.js"
```

Comment:
Missing role name, missing rest of the setup. Needs review and work from @Mastagoon.